jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/objectActions/DownloadObjectAction.kt

```diff
@@ -34,28 +41,114 @@ class DownloadObjectAction(private val project: Project, treeTable: S3TreeTable)
         }
     }
 
-    override fun enabled(nodes: List<S3TreeNode>): Boolean = nodes.all { it is S3TreeObjectNode }
+    private fun downloadSingle(project: Project, file: S3TreeObjectNode) {
+        val destination = getDownloadLocation(false) ?: return
```

Comment:
nit: named parameters make sense here IMO

Reply:
Agree, will add it